### PR TITLE
Us 03 03

### PR DESCRIPTION
**Pull request by lamartm**

### CODE REVIEW

File: Homepage/script.js
Comment (MorrisRoozen) [inline Homepage/script.js:1]: Name should be more specific than script.js

File: styles.css [hunk 1]
```diff
@@ -5,4 +5,60 @@
 
 body {
     font-family: Arial, Helvetica, sans-serif;
+}
+
+/* Recommendation Slider */
```
Comment (MorrisRoozen) [inline styles.css:10]: If this styling is only used in the homepage, move to something like homepage.css

**Review by MorrisRoozen — CHANGES_REQUESTED** [review]

**Review by Tatjanae — COMMENTED** [review]
Only thing that you might (want) to change is to make a separate file for the Javascript and CSS for your carousel and put it in a separate folder (so slick.css and slick.js).

File: Homepage/homepage.html [hunk 2]
```diff
@@ -3,8 +3,44 @@
...
+            <p>€39.00</p>
+        </article>
+        <article class="recommendation-wrapper">
+            <img class="recommendation-image" src="../resources/vase.jpg" alt="vase">
+            <h3>bormla</h3>
+            <p  class="recommendation-description">Lorem ipsum dolor sit amet consectetur, adipisicing elit.</p>
+            <p>€39.00</p>
+        </article>
+    </section>
+<script src="https://code.jquery.com/jquery-3.7.1.js" integrity="sha256-eKhayi8LEQwp4NKxN+CfCh+3qOVUtJn3QNZ0TciWLP4=" crossorigin="anonymous"></script>
```
Comment (MorrisRoozen) [inline Homepage/homepage.html:42]: If these scripts can be at the top of the file, like in the head, they should be

File: Homepage/homepage.html [hunk 3]
```diff
@@ -3,8 +3,44 @@
...
+        <article class="recommendation-wrapper">
+            <img class="recommendation-image" src="../resources/vase.jpg" alt="vase">
+            <h3>bormla</h3>
+            <p class="recommendation-description">Lorem ipsum dolor sit amet consectetur, adipisicing elit.</p>
+            <p>€39.00</p>
+        </article>
+        <article class="recommendation-wrapper">
+            <img class="recommendation-image" src="../resources/vase.jpg" alt="vase">
+            <h3>bormla</h3>
+            <p  class="recommendation-description">Lorem ipsum dolor sit amet consectetur, adipisicing elit.</p>
```
Comment (MorrisRoozen) [inline Homepage/homepage.html:20]: double space between '<p' and 'class' 
We should probably look into getting a linter

**Review by MorrisRoozen — APPROVED** [review]

**Review by Tatjanae — APPROVED** [review]

**Pull request merged**